### DISCUSSION TITLE
Can't turn off CoPilot

### DISCUSSION THREAD

**Discussion by foxydevloper** [community/community#50493] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Please for the love of god let me turn off copilot. I haven't been able to turn off copilot for a month, I have to disable the extension and  restart vscode whenever i want to disable it. The button does nothing

I'm using windows 10, vscode 1.76.2, copilot extension version v1.78.9758. This bug has been here for like a month

https://user-images.githubusercontent.com/12120644/226336290-9c59971d-ac4a-4d26-89cb-c0b98e2ae22f.mp4

**Comment by Akash1134** [comment]
Hi @foxydevloper  i'm sorry you are having difficulties. Would you mind checking this [discussion](https://github.com/community/community/discussions/10930) and let us know if that helps?

**Reply by Salm3** [reply]
[](url)

**Comment by Kiasti** [comment]
This is also still happening on vs2022. wont deactivate. Cant swap the tab hotkey

**Reply by Salm3** [reply]
- [``]( @ #
/)

**Comment by vincentkg** [comment]
happening for me aswell. I'd love to be able to quickly turn it on or off via hotkey, or have the ability to get a suggestion only when I specifically want it and for nothing else
